data/templates/new_game_template

```diff
@@ -0,0 +1,18 @@
+[global]
+interactive_player="1"
+win_condition="scripting/win_conditions/endless_game.lua"
+peaceful="false"
+custom_starting_positions="false"
+map="maps/Lake_of_Tranquility.wmf"
+
+[player_1]
+tribe=
```

Comment:
Why is the `tribe` empty? What happens if it is empty?

In general this file should have a lot of comments, imho. I think most players do not understand each option.

Reply:
Added a lot of explanatory comments. Empty tribe means Random.